### PR TITLE
[#3799]feat(doc): Add documents to note to users the limits of OpenCSVSerde

### PR DESCRIPTION
**Pull request by theoryxu**
### What changes were proposed in this pull request?

Add documents to note to users the limits of OpenCSVSerde in spark-catalog-hive.md

### Why are the changes needed?

Fix: #3799

### CODE REVIEW

**Comment by theoryxu** [comment]
@FANNG1 @jerryshao please help to review, Thanks.

File: docs/spark-connector/spark-catalog-hive.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Supports most DDL and DML operations in SparkSQL, except such operations:
 - `CREATE TABLE LIKE` clause
 - `TRUCATE TABLE` clause
 
+
+:::info
+There are some limits for tables whose row format serde is `org.apache.hadoop.hive.serde2.OpenCSVSerde`. Due to HIVE-13709, the table's scheme may mismatch between Gravitino and Hive Metastore. Please make sure all fields are string type if used.
```
Comment (FANNG1) [inline docs/spark-connector/spark-catalog-hive.md:24]: simple the description for end user?  like Doesn't support read write read write tables with `org.apache.hadoop.hive.serde2.OpenCSVSerde` row format?

Comment (theoryxu) [reply docs/spark-connector/spark-catalog-hive.md]: done

**Review by FANNG1 — APPROVED** [review]

**Comment by FANNG1** [comment]
ci failures, wait for #4340 merged

**Pull request merged**